### PR TITLE
suggested changes

### PR DESCRIPTION
**Pull request by sabrinatoro**

### CODE REVIEW

**Comment by sabrinatoro** [comment]
@nicolevasilevsky I couldn't figure out how to edit your PR in a way that you would see what changes i suggested. 
(you will probably have to merge this PR to yours, then yours to master). 

The biggest changes are: 
1) adding a little bit more explanation
2) I change a lot of the  "Request to remove a term from the obsoletion list" section
I made notes in here with explanations why I think these changes are needed.

File: docs/editors-guide/curation-workflow-overview.md [hunk 1]
```diff
@@ -9,38 +9,39 @@ This document describes the community curation workflow in Mondo, including how
...
   
 ### How to request new terms in Mondo
-1. New term requests should go on our [GitHub tracker](https://github.com/monarch-initiative/mondo/issues).
+1. New term requests should go on our [GitHub tracker](https://github.com/monarch-initiative/mondo/issues). 
     - Guidelines on how to make new term requests are available [here](c-make-good-term-request.md)
 2. Alternatively, you may contact us by email or via Slack to inquire about adding a new term. Please email info@monarchinitative.org with questions or to request to be added to our Slack channel.
 3. Users are always welcome to join our weekly Curation calls,  please email info@monarchinitative.org and request to be added to the invitation.
+If should be noted that every new term request will be represented as a GitHub issue. This allows the community to review and comment on these proposed new terms, and offer guidance on classification, definition,... This open process ensures transparency and representation of the disease view of the community.
```
Comment (nicolevasilevsky) [inline docs/editors-guide/curation-workflow-overview.md:16]:
```suggestion
If should be noted that every new term request will be represented as a GitHub issue. This allows the community to review and comment on these proposed new terms, and offer guidance on classification, definition, etc... This open process ensures transparency and representation of the disease view of the community.
```

File: docs/editors-guide/curation-workflow-overview.md [hunk 2]
```diff
@@ -9,38 +9,39 @@ This document describes the community curation workflow in Mondo, including how
...
-    - Terms are open for community comment for at least two weeks.
-- **Provisional terms**: New terms are added to Mondo by the Mondo Curation team on a [Pull Request (PR)](https://github.com/monarch-initiative/mondo/pulls). Pull Requests undergo automatic quality control checks and are reviewed by members of the Mondo Curation team to check for accuracy, scientific soundness and other potential mistakes that are not identified by our automatic checks. Pull Requests are publicly available and anyone can comment on an open PR to request changes or make comments.
+    - GitHub issues for New Term Request are open for community comment for at least two weeks.
+- **Provisional terms**: New terms are added to Mondo by the Mondo Curation team on a [Pull Request (PR)](https://github.com/monarch-initiative/mondo/pulls). Pull Requests undergo automatic quality control checks and are reviewed by members of the Mondo Curation team to check for accuracy, scientific soundness and other potential mistakes that are not identified by our automatic checks. Similar to GitHub issues, Pull Requests are publicly available and anyone can request changes or make comments on an open PR.
```
Comment (nicolevasilevsky) [inline docs/editors-guide/curation-workflow-overview.md:22]:
```suggestion
- **Provisional terms**: New terms are added to Mondo by the Mondo Curation team on a [Pull Request (PR)](https://github.com/monarch-initiative/mondo/pulls). Pull Requests undergo automatic quality control checks and are reviewed by members of the Mondo Curation team to check for accuracy, scientific soundness and other potential mistakes that are not identified by our automatic checks. Similar to GitHub issues, Pull Requests are publicly viewable and anyone can request changes or make comments on an open PR.
```

**Review by nicolevasilevsky — APPROVED** [review]
thanks @sabrinatoro!

**Pull request merged**